diff -urN chakra-quick-usb-formatter/quickusbformatter.desktop chakra-quick-usb-formatter-patched/quickusbformatter.desktop
--- chakra-quick-usb-formatter/quickusbformatter.desktop	2012-12-01 09:36:59.000000000 +1100
+++ chakra-quick-usb-formatter-patched/quickusbformatter.desktop	2012-12-17 21:49:07.078912128 +1100
@@ -11,4 +11,5 @@
 Name[es]=Formatear USB
 Name[fr]=Formater USB
 Name[gl]=Formatar
-
+Name[ru]=Форматирование USB
+Categories=System;
diff -urN chakra-quick-usb-formatter/quickusbformatter_solid.desktop chakra-quick-usb-formatter-patched/quickusbformatter_solid.desktop
--- chakra-quick-usb-formatter/quickusbformatter_solid.desktop	2012-12-01 09:36:59.000000000 +1100
+++ chakra-quick-usb-formatter-patched/quickusbformatter_solid.desktop	2012-12-17 21:49:11.275920348 +1100
@@ -10,6 +10,7 @@
 Name[es]=Formatear USB
 Name[fr]=Formater USB
 Name[gl]=Formatar
+Name[ru]=Форматирование USB
 
 [Desktop Entry]
 Actions=open;
diff -urN chakra-quick-usb-formatter/translations/ru.po chakra-quick-usb-formatter-patched/translations/ru.po
--- chakra-quick-usb-formatter/translations/ru.po	2012-12-01 09:36:59.000000000 +1100
+++ chakra-quick-usb-formatter-patched/translations/ru.po	2012-12-17 21:44:30.098370689 +1100
@@ -94,8 +94,8 @@
 #. i18n: file: window.ui:273
 #. i18n: ectx: property (text), widget (QPushButton, but_format)
 #: build/ui_window.h:322 translations/rc.cpp:41 rc.cpp:41
-msgid "Apply Format"
-msgstr "Применить форматирование"
+msgid "Apply format"
+msgstr "Форматировать"
 
 #. i18n: file: window.ui:293
 #. i18n: ectx: property (text), widget (QPushButton, but_exit)
